rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -973,7 +990,9 @@ void main_window::HandlePackageInstallation(QStringList file_paths)
 		}
 	}
 
-	if (worker())
```

Comment:
You do not need a value keeper, if you call worker.operator()() again it would yeild the same result.

Reply:
The motivation here was that the code was not particularly self explanatory.
Having a variable telling you what this worker call means makes the code much easier to read